pymc_marketing/clv/models/gamma_gamma.py

```diff
@@ -119,14 +117,13 @@ def expected_new_customer_spend(self) -> xarray.DataArray:
 
         return mean_spend
 
+    # TODO: Why isn't this in the child GammaGammaModel class?
```

Comment:
So it can be used by both child classes?

Reply:
Just noticed `GammaGammaIndividual` has an extra `_summarize_mean_data` step that must be called, so it makes sense now.